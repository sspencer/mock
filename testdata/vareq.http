### Delete equal loaf
# @delay=500ms
# @status=200
DELETE /bread/2
