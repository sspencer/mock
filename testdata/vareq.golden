Delete equal loaf: DELETE /bread/2 delay=500ms status=200 header=36 body=0